src/app/dataset-view/additional-components/dataset-settings-component/tabs/scheduling/dataset-settings-scheduling-tab.component.html

```diff
@@ -18,6 +17,10 @@ <h4>Root polling source</h4>
                 aria-label="Select an option"
                 color="primary"
             >
+                <div class="d-flex align-items-center mt-4">
+                    <input class="checkbox-uncacheable me-3" type="checkbox" formControlName="fetchUncacheable" />
+                    <span class="d-inline-block">Fetch uncachable</span>
+                </div>
```

Comment:
It's a little strange this became a top option. It's quite a rare detail, so could be below the main set of options.

Reply:
Fixed